website/cue/reference/components/sources/internal_metrics.cue

```diff
@@ -326,6 +326,12 @@ components: sources: internal_metrics: {
 			default_namespace: "vector"
 			tags:              _component_tags
 		}
+		buffer_send_duration_max_seconds: {
+			description:       "The maximum duration spent sending a payload to this buffer during a two second window."
```

Comment:
I think this needs to be rewritten now, because it's a full-fledged histogram, not just tracking the maximum observed duration.